tests/test-cases.js

```diff
@@ -1804,6 +1804,42 @@ var CUSTOM_TESTS = [
 			});
 		}
 	},
+	{
+		description: "Data is correctly parsed with steps (headers)",
+		expected: [{One: 'A', Two: 'b', Three: 'c'}, {One: 'd', Two: 'E', Three: 'f'}],
+		run: function(callback) {
+			var data = [];
+			Papa.parse('One,Two,Three\nA,b,c\nd,E,f', {
+				header: true,
+				step: function(results) {
+					data.push(results.data);
+				},
+				complete: function() {
+					callback(data);
+				}
+			});
+		}
+	},
+	{
+		// The worker code path does work,
```

Comment:
I do not understand what is this comment talking about. 

The tests are executed under mocha-chorme and I see this [is psasing on travis.](https://travis-ci.org/mholt/PapaParse/jobs/484479176#L901)

Reply:
That comment was made while the test was failing... and I didn't realize there was a chrome headless test ...i will remove it.